docs/LV_client/tools/mono_parse_binaries.md

```diff
@@ -0,0 +1,21 @@
+# mono_parse_binaries.vi
+
+<p class="img_container">
+<img class="lg_img" src="../mono_parse_binaries.png"/>
+</p>
+
+### Description
+
```

Comment:
Just a few small things here. "Read and parse a binary file according to the type of sensor. 
This VI parses IMU, RPM , GPS, Camera and LiDAR binary files created with the mono_logger.vi." Added "a" before binary file, added "s" to parses, and a period at the end of the paragraph.